Sources/StreamChatUI/Utils/AudioPlayer/AudioPlaybackState.swift

```diff
@@ -0,0 +1,37 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+// Defines an enumeration for audio player playback state
+public enum AudioPlaybackState: Equatable, CaseIterable {
+    // Cases representing different playback states
+    case notLoaded
+    case loading
+    case paused
+    case playing
+    case stopped
+
+    // Equatable implementation for comparing two instances of
+    // AudioPlayerPlaybackState
+    public static func == (
```

Comment:
is this really needed? Also, the doc says `AudioPlayerPlaybackState`, shouldn't it be `AudioPlaybackState`?

Reply:
just a remnant that needed clear up